vllm/engine/output_processor/stop_checker.py

```diff
@@ -57,7 +57,8 @@ def maybe_stop_sequence(
         # Check if a stop token was encountered.
         # This assumes a single token produced per step.
         last_token_id = seq.get_last_token_id()
-        if last_token_id in sampling_params.stop_token_ids:
+        if sampling_params.stop_token_ids and \
+            last_token_id in sampling_params.stop_token_ids:
```

Comment:
nit: bit simpler
```suggestion
        if last_token_id in (sampling_params.stop_token_ids or ()):
```

Reply:
good suggestion, thanks!

Reply:
Maybe add a method to `sampling_params` like `has_stop_token`? (Although this seems to only be used here...)

Reply:
true, but it would read nicer in this spot! :-)

Reply:
I applied the suggestion from @njhill here